test/unit/user_test.rb

```diff
@@ -231,23 +231,25 @@ def setup_user(operation)
   test "non-admin user can delegate roles he has assigned already" do
     setup_user "create"
     create_role          = Role.find_by_name 'create_users'
-    record               = User.new :login    => "dummy", :mail => "j@j.com", :auth_source_id => AuthSourceInternal.first.id,
-                                    :role_ids => [create_role.id.to_s]
+    record               = User.new :login    => "dummy", :mail => "j@j.com", :auth_source_id => AuthSourceInternal.first.id
     record.password_hash = "asd"
     assert record.valid?
```

Comment:
It'd be safe to move these `assert record.valid?` and `assert record.save` below line 238 and remove the assertion introduced here. What do you think about removing the `assert record.valid?` even, since `record.save` already calls `.valid?`? Something like:
```ruby
  record.password_hash = "asd"
  record.role_ids = [create_role.id.to_s]
  assert record.save
  refute record.new_record?
```